pkg/resiliency/resiliency.go

```diff
@@ -532,6 +536,39 @@ func (r *Resiliency) EndpointPolicy(ctx context.Context, app string, endpoint st
 				}
 			}
 		}
+	} else {
+		if defaultNames, ok := r.getDefaultPolicy(Endpoint); ok {
+			r.log.Debugf("Found Default Policy for Endpoint %s: %+v", app, defaultNames)
```

Comment:
Please, log info since this can help a lot with troubleshooting when users/customers report an issue.

Reply:
I think that'd be a bit verbose. It would be displayed at every call since policies are loaded at call time, not stored in memory permanently. Here's that same log as it exists today:

https://github.com/dapr/dapr/blob/master/pkg/resiliency/resiliency.go#L506